jekyll/_cci2/persist-data.md

```diff
@@ -95,73 +99,72 @@ Details about individual step storage and network transfer usage can be found in
 
 ![storage-usage-overage]( {{ site.baseurl }}/assets/img/docs/storage-usage-overage.png)
 
 #### Network
 {: #network }
 {:.no_toc}
 
-To calculate monthly network costs from your usage, click on the Objects tab to see if your organization has accrued any overages (your network egress). Your overage GB can be mutliplied by 420 credits to estimate the total montly costs.
+To calculate monthly network costs from your usage, click on the Objects tab to see if your organization has accrued any overages (your network egress). Your overage GB can be multiplied by 420 credits to estimate the total monthly costs.
 
 The GB allotment only applies to outbound traffic from CircleCI. Traffic within CircleCI is unlimited.
 
 ![network-usage-overage]( {{ site.baseurl }}/assets/img/docs/network-usage-overage.png)
 
-### How to manage your storage and network transfer use
-{: #how-to-manage-your-storage-and-network-transfer-use }
+### How to optimize your storage and network transfer use
+{: #how-to-optimize-your-storage-and-network-transfer-use }
 
 There are several common ways that your configuration can be optimized to ensure you are getting the most out of your storage and network usage.
 
-Before attempting to reduce data usage, you should first consider whether that usage is providing enough value to be kept. In the cases of caches and workspaces this can be quite easy to compare - does the developer/compute time saving from the cache outweigh the cost of the download and upload? Please see below for examples of storage and network optimization opportunities.
-
-### How to manage your storage and network transfer use
-{: #how-to-manage-your-storage-and-network-transfer-use }
-
-There are several common ways that your configuration can be optimized to ensure you are getting the most out of your storage and network usage.
+For example, when looking for opportunities to reduce data usage, consider whether specific usage is providing enough value to be kept.
 
-Before attempting to reduce data usage, you should first consider whether that usage is providing enough value to be kept. In the cases of caches and workspaces this can be quite easy to compare - does the developer/compute time saving from the cache outweigh the cost of the download and upload? Please see below for examples of storage and network optimization opportunities.
+In the cases of caches and workspaces this can be quite easy to compare - does the developer or compute time-saving from the cache outweigh the cost of the download and upload?
 
-### Opportunities to reduce artifact and cache/workspace traffic
-{: #opportunities-to-reduce-artifact-and-cacheworkspace-traffic }
+Please see below for examples of storage and network optimization opportunities through reducing artifact, cache, and workspace traffic.
```

Comment:
We try not to say "please". Maybe: "The sections below show examples of storage ..."